src/v2/components/FireList.jsx

```diff
@@ -89,6 +90,25 @@ export default function FireList(props) {
     }
   }, [nFires, selectedIndex])
 
+  const handleVoteForFire = useCallback((index, vote) => {
+    if (index > -1 && index < nFires) {
+      const fire = fires[index]
+      const {cameraID, timestamp} = fire
+      const endpoint = vote === 'undo' ? '/api/undoVoteFire' : '/api/voteFire'
+      const isRealFire = vote === 'yes'
+
+      query.post(endpoint, {cameraID, isRealFire, timestamp}).then(() => {
+        if (vote === 'undo') {
+          delete fire.voted
+        } else {
+          fire.voted = vote === 'yes'
```

Comment:
You could use isRealFire vs "vote === 'yes'"

Reply:
Okay (was copying the code from VoteFires.js)